specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -27,7 +27,7 @@ options:
     enable-sync-stack: true
     generate-tests: false
     custom-types-subpackage: "implementation.models"
-    custom-types: "FunctionCallModelBase,FunctionCallPreset,FunctionCallPresetFunctionCallModel,FunctionNameFunctionCallModel"
+    custom-types: "ImageGenerationResponseFormat"
```

Comment:
curious about the reason for removing `FunctionCall` custom types.

Reply:
In 0.8.3, the "internal" (or access=internal) operation will only generate procotol API (without models). <-- this applies to `getChatCompletionsWithAzureExtensions` operation which contains the `FunctionCall` classes.

In 0.9.0, codegen augmented to generate "internal" operation as "package private", and its models (even that with Union) to "implementation.models" package. Therefore, marking `getChatCompletionsWithAzureExtensions` as "internal" automatically make these models generated into "implementation.models" package.